rest-api/src/main/scala/za/co/absa/enceladus/rest_api/SpringFoxConfig.scala

```diff
@@ -39,11 +39,18 @@ class SpringFoxConfig extends ProjectMetadata {
   }
 
   private def filteredPaths: Predicate[String] =
-    or[String](regex("/api/dataset.*"), regex("/api/schema.*"),
+    or[String](
+      // api v2
+      regex("/api/dataset.*"), regex("/api/schema.*"),
       regex("/api/mappingTable.*"), regex("/api/properties.*"),
-      regex("/api/monitoring.*"),regex("/api/runs.*"),
+      regex("/api/monitoring.*"), regex("/api/runs.*"),
       regex("/api/user.*"), regex("/api/spark.*"),
-      regex("/api/configuration.*")
+      regex("/api/configuration.*"),
```

Comment:
I would not document v2 API as we do not want people to use it. Thoughts?

Maybe spring fox could have multiple profiles? Dev and non-dev

Reply:
I agree with not documenting v2

Reply:
So, I have changed the implementation to the following: 
 - JVM being supplied `-Dspring.profiles.active=dev` will result in full V2 + V3 API as before, 
 - by default (no special profile setup) will only allow Swagger to show API V3 + import/export methods of API v2 (these we consider public API in V2 already).

Hope this is satisfactory.

### Showcase DEV `-Dspring.profiles.active=dev`
![menas-api-dev](https://user-images.githubusercontent.com/4457378/167129720-561e0bb9-7b4c-49d0-a945-557a989ac673.png)


### Showcase Non-DEV
![menas-api-nondev](https://user-images.githubusercontent.com/4457378/167129704-a960e2e4-0a88-4cf5-b545-3f4b641bc8ae.png)

Reply:
Redone due to https://github.com/AbsaOSS/enceladus/pull/2046#discussion_r867700135.
Menas DEV API remains unchanged, for Menas non-DEV API now only contains V3 API endpoints (i.e. V2 import/export removed)